packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -40,10 +46,7 @@ export enum PaymentMethod {
   BANK = 'BANK',
   EXCHANGE = 'EXCHANGE',
   ADDRESS = 'ADDRESS',
-  PONTO = 'PONTO',
-  KOTANI = 'KOTANI',
-  BITFY = 'BITFY',
-  FLOW_BTC = 'FLOW_BTC',
+  LocalProvider = 'LocalProvider',
```

Comment:
I assume we don't want to change the rest of these enums to the camel case format to avoid messing up analytics?

Reply:
That was my original thinking but on second thought I think we should just make the change now. Probably better to start uniformly enforcing the eng standards we want and deal with the analytics consequences by accounting for it in our scripts